pkg/openstack/nova.go

```diff
@@ -302,7 +309,12 @@ func ReconcileNova(ctx context.Context, instance *corev1beta1.OpenStackControlPl
 		Log.Info(fmt.Sprintf("Nova %s - %s", nova.Name, op))
 	}
 
-	if nova.IsReady() {
+	if nova.IsReady() { //FIXME ObservedGeneration
```

Comment:
observedGeneration is available since https://github.com/openstack-k8s-operators/nova-operator/commit/1559023133be0a6f248b7992414fa5a2dc74d75d

Reply:
Sure, lets do this in a follow up